ensureconda/__init__.py

```diff
@@ -1,4 +1,9 @@
-__version__ = "1.3.0"
-
+import pkg_resources
 
 from .api import ensureconda
+
+
+try:
+    __version__ = pkg_resources.get_distribution("conda_lock").version
```

Comment:
Ideally we should move away from `pkg_resources` but this is fine for now/

Reply:
Spotted this should be `ensureconda`